File: 1-Iniciante/ex1172.c
#include <stdio.h>

int main(){
    /*variaveis*/
    int x[10], i;

    /*entrada*/
    for(i = 0; i < 10; i++){
        scanf("%d", &x[i]);
    }

    /*processamento*/
    for(i = 0; i < 10; i++){
        if(x[i] <= 0){
            x[i] = 1;
        }
    }

    /*saida*/
    for(i = 0; i < 10; i++){
        printf("X[%d] = %d\n", i, x[i]);
    }

    return 0;
}

File: 1-Iniciante/Ex1172.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Ex1172 {
    public static void main(String[] args) throws IOException {
        // variaveis & objetos
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        // entrada & processamento & saida
        int[] vetor = new int[10];

        for (int i = 0; i < 10; i++) {
            int x = Integer.parseInt(br.readLine());
            if (x <= 0) {
                vetor[i] = 1;
            } else {
                vetor[i] = x;
            }
            System.out.printf("X[%d] = %d\n", i, vetor[i]);
        }
        br.close();
    }
}